{"name": "Проверка создания курьера с логином, который уже есть", "status": "passed", "description": "Проверка, что при создании курьера, который уже есть, возвращается статус 409 и сообщение \"Этот логин уже используется. Попробуйте другой.\"", "start": 1727614166371, "stop": 1727614166680, "uuid": "b9ea546c-74d7-4f37-8816-feae7f3f973d", "historyId": "84dba8b03acba865e4ab8a4818ff2a8e", "testCaseId": "84dba8b03acba865e4ab8a4818ff2a8e", "fullName": "tests.test_creat_courier.TestCreateCourier#test_create_courier_with_old_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_creat_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Pro--Polanski.local"}, {"name": "thread", "value": "44343-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_creat_courier"}]}